asreview/entry_points/simulate.py

```diff
@@ -171,11 +172,12 @@ def _simulate_parser(prog="simulate", description=DESCRIPTION_SIMULATE):
              f"Default {DEFAULT_N_INSTANCES}.")
     parser.add_argument(
         "--n_queries",
-        type=int,
+        type=type_n_queries,
         default=None,
-        help="The number of queries. By default, the program "
-             "stops after all documents are reviewed or is "
-             "interrupted by the user."
+        help="The number of queries. Alternatively, entering 'min' will stop the "
+             "simulation when all relevant documents have been found. By default, "
```

Comment:
Can you change ' documents'  into 'records'? See our vocabulary: https://asreview.readthedocs.io/en/latest/intro/vocabulary.html